MIGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`). Compatible with Javascript and TypeScript.
```

Comment:
Let's use package for both libraries, and the legacy package is named `@cloudant/cloudant`
```suggestion
This document is to assist in migrating from the `nodejs-cloudant` (package: `@cloudant/cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`). Compatible with Javascript and TypeScript.
```

Reply:
Changed in c706430